src/EnergyPlus/HVACInterfaceManager.cc

```diff
@@ -147,34 +146,49 @@ namespace HVACInterfaceManager {
 		// from the outlet of one side of the loop get transfered directly
 		// to the inlet node of the corresponding other side of the loop.
 
-		// REFERENCES:
-		// na
-
-		// Using/Aliasing
 		using DataLoopNode::Node;
 		using namespace DataConvergParams;
 		using DataContaminantBalance::Contaminant;
 
-		// Locals
-		// SUBROUTINE ARGUMENT DEFINITIONS:
-
-		// SUBROUTINE PARAMETER DEFINITIONS:
-
-		// INTERFACE BLOCK SPECIFICATIONS
-		// na
-
-		// DERIVED TYPE DEFINITIONS
-		// na
-
-		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
 		static Array1D< Real64 > TmpRealARR( ConvergLogStackDepth ); //Tuned Made static
 		Real64 DeltaEnergy;
-		// FLOW:
+
+		if ( ( CalledFrom == CalledFromAirSystemDemandSide ) && ( OutletNode == 0 ) ) {
+			// Air loop has no return path - only check mass flow and then set return inlet node mass flow to sum of demand side inlet nodes
+			AirLoopConvergence( AirLoopNum ).HVACMassFlowNotConverged( 1 ) = false;
+			AirLoopConvergence( AirLoopNum ).HVACHumRatNotConverged( 1 ) = true;
+			AirLoopConvergence( AirLoopNum ).HVACTempNotConverged( 1 ) = true;
+			AirLoopConvergence( AirLoopNum ).HVACEnergyNotConverged( 1 ) = true;
+			AirLoopConvergence( AirLoopNum ).HVACEnthalpyNotConverged( 1 ) = true;
+			AirLoopConvergence( AirLoopNum ).HVACPressureNotConverged( 1 ) = true;
```

Comment:
I understand only checking for mass flow rate below and setting the flag to true if out of tolerance, but these other flags are initialized to true. Won't that cause reporting for these variables if max iterations are exceeded? Why not set the other flags to false to show they are converged (even though they are not checked below) because it doesn't matter for this case?